block.go

```diff
@@ -127,14 +127,12 @@ func (b *BlockHeader) Hash(api API) (Identifier, error) {
 	return blake2b.Sum256(headerBytes), nil
 }
 
-func (b *BlockHeader) WorkScore(workScoreStructure *WorkScoreStructure) (WorkScore, error) {
-	// Version, NetworkID, IssuingTime, SlotCommitmentID, LatestFinalisedSlot and IssuerID
-	return workScoreStructure.DataByte.Multiply(serializer.OneByte +
-		serializer.UInt64ByteSize +
-		serializer.UInt64ByteSize +
-		CommitmentIDLength +
-		SlotIndexLength +
-		AccountIDLength)
+func (b *BlockHeader) WorkScore(_ *WorkScoreStructure) (WorkScore, error) {
+	return 0, nil
```

Comment:
Is block header always free?

Reply:
The BlockHeader's data will still be charged for in the highest level object, the protocol block, where the entire size of the protocol block, including block header within, is used to get the data factor of the workscore.